src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+tile.ironchest.shulker_box.crystal.green.name=Vihreä Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.red.name=Punainen Kristallinen Shulker-laatikko
+tile.ironchest.shulker_box.crystal.black.name=Musta Kristallinen Shulker-laatikko
+
+tile.ironchest.shulker_box.obsidian.white.name=Valkoinen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.orange.name=Oranssi Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.magenta.name=Violetti Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.light_blue.name=Vaaleansininen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.yellow.name=Keltainen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.lime.name=Lime Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.pink.name=Pinkki Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.gray.name=Harmaa Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.silver.name=Vaaleanharmaa Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.cyan.name=Syaani Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.purple.name=Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.blue.name=Sininen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.brown.name=Ruskea Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.green.name=Vihreä Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.red.name=Punainen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.black.name=Musta Obsidiaani Shulker-laatikko
+
+item.ironchest.shulker_box.iron_gold.name=Parannus: Rautaisesta Kultaisen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.gold_diamond.name=Parannus: золотого в алмазный Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.copper_silver.name=Parannus: медного в серебряный Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.silver_gold.name=Parannus: серебряного в Kultainen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.copper_iron.name=Parannus: медного в Rautainen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.diamond_crystal.name=Parannus: алмазного в кристальный Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.vanilla_iron.name=Parannus: обычного в Rautainen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.vanilla_copper.name=Parannus: обычного в медный Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.diamond_obsidian.name=Parannus: алмазного в обсидиановый Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+
+##############
+#    GUIs    #
+##############
+
+itemGroup.ironchest=Rautaineniset arkut
```

Comment:
Fix typo

Reply:
Ok

Reply:
If you want, check new file. Now I do it so careful